doc/source/Container.md

```diff
@@ -0,0 +1,132 @@
+# Running watson-conversation-slots-intro in a Container on Bluemix with Kubernetes
+
+This directory allows you to deploy the `watson-conversation-slots-intro` application into a container running on Bluemix, using Kubernetes.
+
+The commands below use environment variables in order to define the specific details of the deployment. Either run the following to export the ENV variables, or substitute your names in the commands or exports:
+```
+$ export CLUSTER_NAME=Watson
+$ export CONVERSATION_SERVICE=conversation-service-watson-pizzeria
+$ export CONFIG_MAP=watson-pizzeria-config
+$ export POD_NAME=watson-pizzeria-pod.yml
+$ export KUBE_SERVICE=pizza-bot
+$ export CONTAINER_ENV_VARIABLE=CONVERSATION_SERVICE_WATSON_PIZZERIA
+```
+
+
+# Steps
+
+## Create the Kubernetes cluster
+
+* Follow the instructions to [Create a Kubernetes Cluster](https://github.com/IBM/container-journey-template).
+```
+$ bx cs cluster-create --name $CLUSTER_NAME
+```
+
+## Create the Watson Conversation Service and bind to your cluster
+
+Either follow the instructions to [Create a Conversation Service](https://console.ng.bluemix.net/catalog/services/conversation) or Perform the following from the Bluemix CLI.
+
+* Create the Watson Conversation service:
+
+```
+$ bx service create conversation free $CONVERSATION_SERVICE
+```
+
+
+* Verify that the service instance is created:
+
+```
+$ bx service list
+```
+
+* Obtain the ID of your cluster:
+
+```
+$ bx cs clusters
+```
+
+* Bind the service instance to your cluster:
+
+```
+$ bx cs cluster-service-bind <cluster-ID> $CONVERSATION_SERVICE
+```
+
+## Load the Watson Conversation
+
+* Follow the instructions to [Load the Watson Conversation](https://github.com/IBM/watson-conversation-slots-intro#4-configure-watson-conversation)
+
+## Create a Kubernetes Configuration Map with the Workspace ID
+
+* Create a Kubernetes Configuration Map:
+
+```
+$ kubectl create configmap $CONFIG_MAP \
+    --from-literal=workspace_id=<WorkSpace_ID>
```

Comment:
WORKSPACE_ID to be consistent w/ the standard env we use there?  Actually why isn't it $WORKSPACE_ID?

Reply:
It can't be $WORKSPACE_ID because we never export it. We don't have it at the top, when we export the vars to turn generic instructions into specific .

Reply:
I've changes so that we do export, and then use the ENV var...